packages/dns-discovery/src/index.ts

```diff
@@ -1,3 +1,3 @@
-export { PeerDiscoveryDns, wakuDnsDiscovery } from "./dns_discovery.js";
```

Comment:
huuuum, I am not convinced we want to add teh `lib` folder everywhere, I was more inclined to actually remove it from `core` package.

Reply:
It just seems unnecessary as js-waku is a library. there is no app vs lib in the code, everything is library code.
I would just create a `utils` folder to segregate orphan tools.